src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -390,9 +427,15 @@ public TreeEnsembleFeaturizerBindableMapper(IHostEnvironment env, ModelLoadConte
 
             // *** Binary format ***
             // ensemble
+            // string: treesColumnName
+            // string: leavesColumnName
+            // string: pathsColumnName
 
             ctx.LoadModel<TreeEnsembleModelParameters, SignatureLoadModel>(env, out _ensemble, "Ensemble");
             _totalLeafCount = CountLeaves(_ensemble);
+            _treesColumnName = ctx.LoadStringOrNull();
```

Comment:
Do we need to bump the version number?

Reply:
It's BC, so maybe no.

Reply:
What does "BC" mean?

Reply:
This module is backward compatible. All previously saved models work as what they were doing.

Reply:
Ah.. No, you're right. Let me bump the version.